### DISCUSSION TITLE
Tautulli for Media Monitoring

### DISCUSSION THREAD

**Discussion by joe-heaton** [tteck/Proxmox#950] [Ideas]
Would compliment the rest of the media services! 

Thanks for everything you do,  coffee donated!

**Comment by sendmebits** [comment]
Looks like this has been added now:


https://tteck.github.io/Proxmox/

`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/tautulli.sh)"`
